reqhandler/request_handler_test.go

```diff
@@ -12,7 +12,19 @@ type RequestHandlerSuite struct{}
 
 var _ = Suite(&RequestHandlerSuite{})
 
-func (s *RequestHandlerSuite) TestHandlesUDPConnection(c *C) {
-	output := HandleUDPConnection(&mockUDPConn{})
-	c.Assert(true, Equals, output)
+func (s *RequestHandlerSuite) TestDataFromUDPConnection(c *C) {
+	udpPacket := []byte("TEST_CONNECTION")
+	m := &mockUDPConn{data: udpPacket}
+	output, err := readRequest(m)
+
+	c.Assert(err, IsNil)
+	c.Assert(true, Equals, bytes.Equal(udpPacket, output[:len(udpPacket)]))
```

Comment:
The ordering here should also be reversed. bytes.Equal(...) should be the first argument and true should be the last. This is also the case for other Asserts as well, except those that use a Checker that only requires one value (like IsNil).